.github/workflows/MainDistributionPipeline.yml

```
@@ -23,8 +23,8 @@ jobs:
 
   duckdb-stable-build:
     name: Build extension binaries
-    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.3.1
+    uses: ./.github/workflows/_extension_distribution.yml
     with:
       duckdb_version: v1.3.1
-      ci_tools_version: v1.3.1
+      ci_tools_version: v1.3.2
       extension_name: fastlanes
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)